index.html

```diff
@@ -7,19 +7,17 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Layout Practice Revisited</title>
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
     <link rel="stylesheet" href="css/styles.css">
 </head>
 
 <body>
     <div class="container-fluid">
         <div class="container">
             <div class="row top">
-                <div class="col-sm-2 col-md-2 brand">Brand</div>
-                <div class="col-sm-2 col-md-2 col-sm-push-3 col-md-push-3 nav">Push 6</div>
-                <div class="col-sm-2 col-md-2 col-sm-push-3 col-md-push-3 nav">Push 8</div>
-                <div class="col-sm-2 col-md-2 col-sm-push-3 col-md-push-3 nav">Push 10</div>
+                <div class="col-xs-sm-2 col-sm-2 col-md-2 brand">Brand</div>
+                <div class="col-xs-sm-2 col-sm-2 col-md-2 col-xs-push-3 col-sm-push-3 col-md-push-3 nav">Push 6</div>
```

Comment:
If you're using `push` columns, you have to have `pull` columns in the same row to swap with.